packages/frontend/src/components/Stats/Stats.tsx

```diff
@@ -0,0 +1,22 @@
+import { useAllTimeStats } from 'src/hooks/useAllTimeStats';
+import { numberWithCommas } from 'src/utils/numberWithCommas';
+
+const Stats = () => {
+  const { isLoading, error, data } = useAllTimeStats();
+
+  if (isLoading) return <div className="text-xs invisible p-2">Loading all-time volume...</div>;
+
+  if (error) {
+    console.error('Failed to fetch all-time stats:', error);
+
+    return null;
+  }
+
+  return (
+    <div className="font-display m-auto text-center text-xs p-2 flex place-items-center justify-center relative">
+      All-time volume: {data && `$${numberWithCommas(data)}`}
```

Comment:
Tiny on desktop, had to put my glasses on

Reply:
Good point, only tested on giant monitors on desktop!